app/api/get-daily/route.ts

```
@@ -28,7 +28,10 @@ export async function GET() {
 
   const dailyReflection = await getOrCreateDailyReflection();
 
-  dailyReflection.book.imageUrl = dailyReflection.book.media.url;
+  if (dailyReflection.book && dailyReflection.book.imageUrl) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)